Source/RageCommunity.Library.TestPlugin/EntryPoint.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Rage;
+
+[assembly: Rage.Attributes.Plugin("Rage Community Library Test", Description = "A plugin to test the contents of the Rage Community Library.", Author = "NightYield and Contributors")]
```

Comment:
I recommend adding `PrefersSingleInstance = true` after the Author attribute to prevent the possibility of the plugin being loaded more than once.